src/currencyservice/Dockerfile

```diff
@@ -1,46 +1,59 @@
-# Copyright 2020 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-FROM node:16-alpine as base
-
-FROM base as builder
-
-# Some packages (e.g. @google-cloud/profiler) require additional
-# deps for post-install scripts
-RUN apk add --update --no-cache \
-    python3 \
-    make \
-    g++
-
-WORKDIR /usr/src/app
-
-COPY package*.json ./
-
-RUN npm install --only=production
-
-FROM base
-
-RUN GRPC_HEALTH_PROBE_VERSION=v0.4.7 && \
-    wget -qO/bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 && \
-    chmod +x /bin/grpc_health_probe
-
-WORKDIR /usr/src/app
-
-COPY --from=builder /usr/src/app/node_modules ./node_modules
-
-COPY . .
-
-EXPOSE 7000
-
-ENTRYPOINT [ "node", "--require", "./tracing.js", "server.js" ]
+FROM ubuntu:20.04
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get -y update
+RUN apt-get -y upgrade && apt-get -y dist-upgrade
+RUN apt-get install -qq -y --ignore-missing \
+	build-essential		                    \
+	git			                            \
+	make			                        \
+	pkg-config		                        \
+	protobuf-compiler	                    \
+	libprotobuf-dev                         \
+	sudo			                        \
+	wget			                        \
+    cmake
+
+# The following arguments would be passed from docker-compose.yml
+ARG GRPC_VERSION="1.46.0"
+ARG OPENTELEMETRY_VERSION="1.4.0"
```

Comment:
This comment is not true in the current state, you could copy these args to the currencyservice in the compose file and remove the value set in this file to fix.

Reply:
Thanks, added in docker-compose as args